ts/features/wallet/creditCard/screen/__tests__/CreditCardDetailScreen.test.tsx

```diff
@@ -65,6 +84,22 @@ describe("Test CreditCardDetailScreen", () => {
     const screen = renderDetailScreen(store, creditCard);
     expect(screen.getByTestId("WorkunitGenericFailure")).not.toBeNull();
   });
+  it("does not render idpay initiatives when isIdpayEnabled is false", async () => {
```

Comment:
We should also test the proper rendering of the component if `isIdpayEnabled` is true (and there are initiatives).